analyzer/pkg/dpkg/testdata/dpkg

```diff
@@ -2130,4 +2130,26 @@ Description: Micro string library: shared library
  This package contains the shared library for ustr.
 Homepage: http://www.and.org/ustr/
 
+Package: git
+Status: uninstall ok installed
```

Comment:
I prefer using an actual value. Did you see this value in `Status`? AFAIK, it's like `Status: deinstall ok config-files`.

Reply:
@knqyf263 fixed